inc/Widget/SmailyWidget.php

```diff
@@ -442,74 +442,81 @@ class="widefat"
 				</p>
 
 				<!-- Customize button color and text. -->
-				<div class="row">
-					<div class="column" id="button-color-container">
-						<!-- Add color picker HTML. -->
-						<div>
-							<label for="<?php echo esc_attr( $this->get_field_id( 'submit_button_color' ) ); ?>">
-								<b><?php esc_html_e( 'Button color', 'smaily' ); ?></b>
-							</label>
-							<!-- Jscolor required:false is used to clear the input value. -->
-							<input
-								data-jscolor="{required:false}"
-								class="button-style"
-								<?php
-								if ( $use_site_submit_button_color === true ) :
-									?>
-									disabled<?php endif; ?>
-								id="<?php echo esc_attr( $this->get_field_id( 'submit_button_color' ) ); ?>"
-								name="<?php echo esc_attr( $this->get_field_name( 'submit_button_color' ) ); ?>"
-								type="text"
-								value="<?php echo esc_attr( $submit_button_color ); ?>" />
-						</div>
-						<div class="default-value-checkbox">
-							<input
-								class="smaily-checkbox default_background_color"
-								<?php
-								if ( $use_site_submit_button_color === true ) :
-									?>
-									checked<?php endif; ?>
-								id="<?php echo $this->get_field_id( 'default_background_color' ); ?>"
-								name="<?php echo $this->get_field_name( 'use_site_submit_button_color' ); ?>"
-								type="checkbox"
-								value="1" />
-							<label for="<?php echo $this->get_field_id( 'use_site_submit_button_color' ); ?>" ><?php esc_html_e( 'Use default button color?', 'smaily' ); ?></label>
+				<p>
```

Comment:
Wrapping `div` with `p` won't work, because `div` is not allowed inside `p`. The browser will correct it to:
```html
<p></p>
<div class="row">
    ...
</div>
```

I think reworking the HTML is inevitable, so you would be able to use `p` inside a `div`instead. You could:
- move `id` attribute from `<div class="column">` to `<div class="row">`;
- replace `<div class="column">` with `<p>`;
- replace `<div class="default-value-checkbox">` with `<p>`;

So you would end up with HTML:
```html
<div class="row" id="...">
    <p>...</p>
    <p>...</p>
</div>
```

Which in turn would allow you to remove `.row`, `.column` and `div.default-value-checkbox` CSS rules and replace them with:
```css
.row > p:first-child {
    margin-bottom: 0.5em;
}
.row > p:last-child {
    margin: 0.5em 1px 1em;
}
```